components/event-receiver/org.wso2.carbon.event.receiver.ui/src/main/resources/web/eventreceiver/eventReceiver_details.jsp

```diff
@@ -291,7 +321,10 @@
                         </thead>
                         <tbody>
                         <%
-                            for (EventMappingPropertyDto metaEbProperties : eventReceiverConfigurationDto.getMetaMappingPropertyDtos()) {
+                            EventMappingPropertyDto[] metaMappingDto = eventReceiverConfigurationDto.getMetaMappingPropertyDtos();
+                            if(metaMappingDto != null && metaMappingDto.length > 0){
```

Comment:
Thanks @prabathabey, Changes are done in pull request - https://github.com/wso2/carbon-analytics-common/pull/122/